src/plugins/data-card/data-card-tile.tsx

```diff
@@ -66,6 +67,14 @@ export const DataCardToolComponent: React.FC<ITileProps> = observer((props) => {
   }, [currEditAttrId, currEditFacet, imageUrlToAdd,
```

Comment:
@bacalj I'd guess you could just add `content.attributes` to the dependency list here and that would take care of things. I don't think you need the `useRef` and extra useEffect.

Reply:
@scytacki oh, whoops I missed this comment and then pushed what I have below.  I can try what you said above if you think that makes more sense.